k8s/admin/monitoring/patches/sandbox/cluster-00/prom-operator.yaml

```diff
@@ -0,0 +1,21 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: prom-operator
+  namespace: monitoring
+  annotations:
+    flux.weave.works/automated: "false"
+    flux.weave.works/ignore: "false"
+spec:
+  values:
+    prometheus:
+      ingress: 
```

Comment:
May be this can also go to common ? try to keep the config in patches only if unavoidable.

Reply:
Ingress config is environment specific. Each environment will have its own DNS entry.